src/evergreen/cli/main.py

```diff
@@ -71,9 +71,9 @@ def list_patches(ctx, project, limit):
     fmt = ctx.obj["format"]
     patches = []
     for i, p in enumerate(api.patches_by_project(project)):
-        patches.append(p.json)
-        if limit and i > limit:
+        if limit and i >= limit:
```

Comment:
This was previously in the wrong place, causing the wrong number of results to return.